composer.json

```diff
@@ -9,19 +9,23 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "gedmo/doctrine-extensions": "^2.3",
-        "knplabs/knp-menu-bundle": "^3",
+        "php": "^7.2 || ^8.0",
+        "gedmo/doctrine-extensions": "^2.3 || ^3.0",
+        "knplabs/knp-menu-bundle": "^3.0",
         "symfony/console": "^4.4",
-        "zicht/admin-bundle": "^6.2.1",
-        "zicht/framework-extra-bundle": "^9",
-        "zicht/url-bundle": "^5"
+        "symfony/dependency-injection": "^4.4",
+        "symfony/http-kernel": "^4.4",
+        "symfony/security": "^4.4",
```

Comment:
Een gevolg van deze toevoeging is dit: https://github.com/zicht/hoortoren.nl/pull/5/files

`symfony/security` is een "pack" en vervangt deze 4 packages:

```json
    "replace": {
        "symfony/security-core": "self.version",
        "symfony/security-csrf": "self.version",
        "symfony/security-guard": "self.version",
        "symfony/security-http": "self.version"
    },
```

Er zijn geen 5.x en 6.x tags aanwezig op deze repository, dus bestaat niet voor Sf 5 en 6? Misschien is het beter om 1 van de specifieke packages te "requiren", hoogstwaarschijnlijk moet dat `symfony/security-core` zijn dan